File: official/docs/ruby/current/billing/add-stripe-credit-card.rb
require 'easypost'

client = EasyPost::Client.new(api_key: 'EASYPOST_API_KEY')

payment_method = client.referral_customer.add_credit_card_from_stripe(
  'REFERRAL_USER_API_KEY',
  'seti_...',
  'primary',
)

puts payment_method

File: official/docs/python/current/billing/add-stripe-credit-card.py
import easypost

client = easypost.EasyPostClient("EASYPOST_API_KEY")

payment_method = client.referral_customer.add_credit_card_from_stripe(
    referral_api_key="REFERRAL_CUSTOMER_PROD_API_KEY",
    financial_connections_id="seti_...",
    priority="primary",
)

print(payment_method)
